packages/react/src/components/SegmentedControl/components/Segment/Segment.tsx

```diff
@@ -40,7 +40,10 @@ export const Segment = React.forwardRef((props, forwardedRef) => {
 
   const state = useSegmentedControl();
 
-  const { inputProps } = useRadio(props, state, inputRef);
+  const { inputProps } = useRadio({
+    'aria-label': props.value,
+    ...props
+  }, state, inputRef);
```

Comment:
I am not sure about assigning value to aria-label - Shouldnt this be the label prop insetad?

Reply:
Same question as my above one - shouldn't it be up to the person using this to pass an aria-label?